Four free python resources

martijnsublime opened the pull request:
Added four additional free resources. Three courses focused on Python for data science, one course focused on python programming. 

Thank you for the resource.

clarecorthell commented:
Thanks for adding a PR! The curriculum is an exclusive list of single best resources on each topic. Please let me know if you believe these are better than the existing resources, and why. Thanks!

martijnsublime commented:
No problem. I think these resources are better/more appropriate for the following reasons:

(i) All the resources I added are free. lowering the barrier to entry. This is not true for all of the resources mentioned on the current list.
(ii) The resources I added focus on a combination of short online videos and interactive in-browser coding exercises. This is of course subjective, but I think this leads to a more engaging learning experience, certainly since on top of that the learning experience is supplemented with elements of gamification. 
(iii) Continuous improvement: The added courses are taken by 100 thousands of students to date, and all providers have build-in feedback systems. This makes sure these courses are well tested with very few errors.  

Hope this helps. Let me know in case you have any further questions.
